src/components/venue-icon/index.tsx

```diff
@@ -1,17 +1,26 @@
 import { ComponentText, useTranslation } from '@atb/translations';
 import { MonoIcon, MonoIconProps } from '@atb/components/icon';
+import { TransportModeType } from '@atb/modules/transport-mode';
 
-export type VenuIconProps = {
-  category: FeatureCategory[];
+export type VenueIconProps = {
+  categories?: FeatureCategory[];
```

Comment:
This feels clunky. Initially wanted one property that could be either `FeatureCategory[]` or `TransportModeType[]`, but creating guards to check which type it is was a little comprehensive. Also didn't want to create another mapping function like `mapLocationCategoryToVenueType` only the other way around. 

`TransportModeType` and `VenueIconType` overlaps so I thought maybe we could do it like this.

Reply:
I couldn't see a better approach to this, but @mikaelbr might have a suggestion about this? 🤔

Reply:
Could separate into different components.

Reply:
I see I wrote "creating guards to check which type it is was a little comprehensive", but can't quite remember why. I'll have to check.

Reply:
Implemented union type and type guard.